atomicfu-gradle-plugin/build.gradle

```diff
@@ -14,8 +14,8 @@ if (rootProject.ext.jvm_ir_enabled) {
 // Gradle plugin must be compiled targeting the same Kotlin version as used by Gradle
 kotlin.sourceSets.all {
     languageSettings {
-        apiVersion = "1.4"
-        languageVersion = "1.4"
+        apiVersion = rootProject.ext.kotlin_api_version
```

Comment:
This is a bit too unmaintainable for us -- both in terms of readability (i.e. _why_ the code is written this way) and writability (i.e. where to update the language version).

I suggest making rr build explicit:
```
languageSettings {
    apiVersion = getApiVersionForOverriddenKotlin() ?: "1.4"
    ... same pattern everywhere else...
}
```

This way we know **why** the version can be overridden, while still having the actual default where it is supposed to be